src/main/java/com/google/sps/servlets/startGameServlet.java

```diff
@@ -45,11 +45,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       long quizTime = 0;
 
       // TODO: check if game name is valid (needs user entity and GameUtils merged)
-      // Query query = new Query("User");
+      // Filter getCorrectUser = new FilterPredicate("userID", FilterOperator.EQUAL, givenUser);
+      // Query query = new Query("Game").setFilter(getCorrectUser);
       // PreparedQuery results = datastore.prepare(query);
       // List<Entity> resultsList = results.asList();
```

Comment:
This is really good. We could make it even better by refactoring it out into a function in a utility class (UserUtility?). that way, other people can use this bit of code (because I'm sure it'll be repeated again elsewhere. 

We can then also test it by mocking out the datastore and ensuring things work as expected. Some good test cases would be where the username doesn't exist, where the username exists, and where the given username is blank.